src/main/resources/swagger/api-docs.yaml

```diff
@@ -357,9 +409,6 @@ definitions:
       googleServiceAccount:
```

Comment:
`googleServiceAccount` should be replaced with `clusterServiceAccount` and `notebookServiceAccount` (my bad). I can fix this.